### DISCUSSION TITLE
add a description to line item

### DISCUSSION THREAD

**Discussion by camille-clappaz** [pay-rails/pay#982] [Q&A]
I created a project with ruby ​​on rails and I use jumpstart pro. I want to edit an invoice with the description of the number of credits purchased and not Docxtemplater Saas. 
On the invoice, Docxtemplater Saas corresponds to application_name of my jumpstart.yml file.
![invoice](https://github.com/pay-rails/pay/assets/121098213/33c5cf88-15fe-43df-b95e-0fdbabdf6816)

![line-items](https://github.com/pay-rails/pay/assets/121098213/06268ccf-f379-4605-a418-a1f6e8b86939)

I tried to write it like this in line items but I got this error:
![line-items-error](https://github.com/pay-rails/pay/assets/121098213/69c27b8b-d3ba-4e75-b3f3-5e96427565ee)

**Comment by excid3** [comment]
The line items you pass to Stripe API calls are different from the line items in receipts. You can customize receipts in the `pay.rb` initializer by overriding methods in `ChargeExtensions`

For example, the PDF line items are defined here and you'd override this method to customize:
https://github.com/pay-rails/pay/blob/main/lib/pay/receipts.rb#L24-L49
